src/Registration/Repositories/Entities/RegistrationEntity.php

```diff
@@ -0,0 +1,135 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Registration\Repositories\Entities;
+
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+use VSV\GVQ_API\Common\Repositories\Entities\Entity;
+use VSV\GVQ_API\Registration\Models\Registration;
+use VSV\GVQ_API\User\Repositories\Entities\UserEntity;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="registration")
+ */
+class RegistrationEntity extends Entity
+{
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", name="hash_code", length=22, nullable=false)
+     */
+    private $hashCode;
```

Comment:
@stejes hier kunnen we ook een unique constraint op zetten en eventueel een index. Maar zal wel niet zo intensief zijn om te zoeken :)